ote/src/clj/ote/components/http.clj

```diff
@@ -115,6 +115,14 @@
    :headers {"Content-Type" "application/json+transit"}
    :body (transit/clj->transit data)})
 
+(defn no-cache-transit-response
+  "Return the given Clojure `data` as a Transit response with status code 200."
+  [data]
+  {:status 200
+   :headers {"Content-Type" "application/json+transit"
```

Comment:
There's minor duplication here, perhaps `transit-response` should take an optional map of extra headers to set, which `no-cache-transit-response` could call with a map that would be merged.